### ISSUE TITLE
Twig doesn't work

### ISSUE THREAD

**Issue by Citrullin** [CachetHQ/Cachet#1437]
i use twig in my incident templates but it doesn't work. It output only the twig code
e.g.`{% if {{ app.request.get('trackDomain') }} == 'test.webtrekk.com'|url_encode %}
 Das ist ein Test
{% endif %}`

I'm on HEAD e33f794. i tested it also with deee3b7. It doesn't work. Is there anything i have to do? Config? Shortcut before the twig code start?

**Comment by GrahamCampbell** [comment]
Ping @jbrooksuk.

**Comment by jbrooksuk** [comment]
I'll take a gander.

**Comment by jbrooksuk** [comment]
Your Twig template is actually incorrect for a start and there is also an issue with Cachet and Twig that is being fixed in #1446.

**Comment by jbrooksuk** [comment]
```twig
{% if app.request.get('trackDomain') == 'test.webtrekk.com'|url_encode %} Das ist ein Test {% endif %}
```
